.travis.yml

```diff
@@ -2,4 +2,16 @@ sudo: false
 language: ruby
 rvm:
   - 2.4.3
-before_install: gem install bundler -v 1.16.1
+install:
+  - sudo apt-get update
+  - sudo apt-get -y install apt-utils automake
+  - wget https://archive.apache.org/dist/pulsar/pulsar-2.4.1/DEB/apache-pulsar-client.deb
```

Comment:
did you try changing the version number here from 2.4.1 to 2.6.1 ?

Reply:
Yes, but unfortunately it fails with the same thing:

```
Failure/Error: require 'pulsar/bindings'
LoadError:
  /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.22' not found (required by /usr/lib/libpulsar.so.2.6.1) - /home/travis/build/instructure/pulsar-client-ruby/lib/pulsar/bindings.so
```